redhat-cop/containers-quickstarts
# Fix wrongly named 'generic agent' file

#### What is this PR About?
When deploying jenkins agents, the README states to run:
```
oc process -f ../../.openshift/templates/jenkins-agent-generic-template.yml \
    -p NAME=jenkins-agent-npm \
    -p SOURCE_CONTEXT_DIR=jenkins-agents/jenkins-agent-npm \
    | oc apply -f -
```

however this fails since `jenkins-agent-generic-template.yml` is in fact named `jenkins-slave-generic-template.yml`. 

This PR fixes that issue

#### How do we test this?
Run the code block from the readme and see this now works

cc: @redhat-cop/day-in-the-life
